### DISCUSSION TITLE
Changing the song fail animation on standard

### DISCUSSION THREAD

**Discussion by cj4m** [ppy/osu#15171] [Ideas] [answered]
I was a huge fan of the notes turning red when I failed a song as it felt a lot less defeating than the plain falling notes of stable, but the new red flash is a huge eye sore and shock when you don't have the low health red screen.

Would it be possible to have that red flash on fail to be tied in with the low health animation? it doesn't feel very fitting when you have the low health option disabled, but I do think it adds a cool FPS game feeling when you have it enabled.

**Comment by integerrr** [comment, marked as answer]
the low health red screen is there, it just comes on at much lower hp now (i think, judging from memory)
![osu_2021-10-18_02-50-43](https://user-images.githubusercontent.com/7279624/137657217-0510c89a-a6d4-43f2-ac15-fb2318e7726e.jpg)
![osu_2021-10-18_02-51-18](https://user-images.githubusercontent.com/7279624/137657225-a16bad44-bbe8-4adf-8ab7-9b744e3843a3.jpg)

**Reply by peppy** [reply]
I'll make the existing option cover the new flash as well.

**Reply by cj4m** [reply]
Yeah I know, I have it disabled personally, but when you fail the map the entire screen flashes red which doesn't feel very fitting if you have the low health screen disabled.

edit: peppy's comment popped up when i posted mine, thanks peppy!
